src/public/meeting.ts

```diff
@@ -267,7 +267,7 @@ export namespace meeting {
     if (!callback) {
       throw new Error('[share app content to stage] Callback cannot be null');
     }
-    ensureInitialized(FrameContexts.sidePanel);
+    ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
```

Comment:
Why would a developer want to call shareAppContentToStage while already in the stage?

Reply:
@ydogandjiev 
1. There are apps who want to switch from one url to another and stage will be replaced with new url. 
2. One of the sharing api provides capabilities and app want to know on stage "Am I the current presenter?" This helps app to change the UI based on presenter and attendee.
Co-watch wants this capability.